benchmarks/dynamo/common.py

```diff
@@ -236,13 +273,30 @@ def print_summary(filename):
             pass
 
 
+def tensor_is_on_xla(tensors):
+    if not isinstance(tensors, (tuple, list)):
+        tensors = [tensors]
+    return any(map(lambda x: x.device.type == "xla", tensors))
+
+
 def timed(model, model_iter_fn, example_inputs, times=1, return_result=False):
     synchronize()
```

Comment:
why don't we also sync_multi here if XLA? i think it is going to work fine as currently written, since it is unlikely there is an xla computation running before the first `timed` call.  but it breaks with tradition, the sync here is 'defensive'